main/src/ca/uwaterloo/flix/language/phase/Verifier.scala

```diff
@@ -211,16 +211,14 @@ object Verifier {
 
         case AtomicOp.Is(sym) =>
           val List(t1) = ts
-          check(expected = MonoType.Enum(sym.enumSym))(actual = t1, loc)
           check(expected = MonoType.Bool)(actual = tpe, loc)
 
         case AtomicOp.Tag(sym) =>
           val List(t1) = ts
-          check(expected = MonoType.Enum(sym.enumSym))(actual = tpe, loc)
+          tpe
```

Comment:
Is there no check we can add?

Reply:
There is, but I wanted to work before spending the effort